src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md

```diff
@@ -55,7 +55,7 @@ It is advised that you save your work on the virtual machine before you run this
 
 Linux: The **VolumeType** parameter is required when encrypting Linux virtual machines, and must be set to a value ("Os", "Data", or "All") supported by the Linux distribution. 
 
-Windows: The **VolumeType** parameter may be omitted, in which case the operation defaults to All; if the VolumeType parameter is present for a Windows virtual machine, it must be set to either All or OS.
+Windows: The **VolumeType** parameter may be omitted, in which case the operation defaults to All; if the VolumeType parameter is present for a Windows virtual machine, it must be set to either All, Data or OS. For encrypting only data disks using the "Data" as volumetype, the OS disk must be encrypted first using "OS" as volumeType.
```

Comment:
This change is unnecessary. It creates confusion by making people think they can have a Windows VM with only Data disks encrypted. I'd leave it as is.

Reply:
It seems you got it wrong. It explicitly states that you need to encrypt OS disk before encrypting the data disks (else the operation fails ).. I think this is making more clear for using the Data as the value of volumetype.